Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,30 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import ActionButton from "./components/ButtonComponents/ActionButton";
+import NumberButton from "./components/ButtonComponents/NumberButton";
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="main-container">
+      <CalculatorDisplay />
+      <NumberButton number="clear" extraStyles="large-btn" />
+      <ActionButton action="&divide;" extraStyles="operation-btn" />
+      <NumberButton number="7" />
+      <NumberButton number="8" />
+      <NumberButton number="9" />
+      <ActionButton action="&times;" extraStyles="operation-btn" />
+      <NumberButton number="4" />
+      <NumberButton number="5" />
+      <NumberButton number="6" />
+      <ActionButton action="&#45;" extraStyles="operation-btn" />
+      <NumberButton number="1" />
+      <NumberButton number="2" />
+      <NumberButton number="3" />
+      <ActionButton action="&#43;" extraStyles="operation-btn" />
+      <NumberButton number="0" extraStyles="large-btn" />
+      <ActionButton action="&#61;" extraStyles="operation-btn" />
```

Comment:
this was an excellent solution! now that you've worked more with map, is there anything you would have done differently?